lib/lightning/kafka_triggers/message_candidate_set_supervisor.ex

```diff
@@ -44,4 +49,52 @@ defmodule Lightning.KafkaTriggers.MessageCandidateSetSupervisor do
       |> Supervisor.child_spec(id: "mcs_worker_#{index}")
     end)
   end
+
+  def generate_child_specs(:message_candidate_set, number_of_workers) do
```

Comment:
If the first parameter is the server module, the worker can be inferred and the function could be reused.

Reply:
@jyeshe I have corrected this in a new PR.